app/controllers/employees/rewards/orders_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Employees
+  module Rewards
+    class OrdersController < EmployeesController
+      before_action :set_reward
```

Comment:
I don't like using callbacks -> [here is why](https://www.visuality.pl/posts/should-i-use-active-record-callbacks). 
What do you think about it?

Reply:
I accept your point of view and i'll try to avoid callback further on